uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1547,69 +1559,75 @@ sub CreateMRICandidates {
     ############################################################
     ### Standardize sex (DICOM uses M/F, DB uses Male/Female)
     ############################################################
-    my $this = shift;
-    my $query = '';
-    my ($subjectIDsref,$sex,$tarchiveInfo,$User,$centerID, $upload_id) = @_;
-    my ($message);
+    my $this  = shift;
+    my ($subjectIDsref, $tarchiveInfo, $User, $centerID, $upload_id) = @_;
+
+    my ($sex, $query, $message);
+    my $dbh    = $this->{dbhr};
+    my $pscID  = $subjectIDsref->{'PSCID'};
+    my $candID = $subjectIDsref->{'CandID'};
+
+
+    # return from the function if createCandidate config setting is not set
+    return if (!NeuroDB::DBI::getConfigSetting($dbh, 'createCandidates'));
```

Comment:
changed a bit the logic here. Checked first if `createCandidates` config setting is set to `true`. If set to `false`, then return from this function candidates should not be created as per the config setting.

Reply:
Good. Since there is a `ConfigOB` that exists now, we should use it to fetch the `createCandidates` setting. This could be done at a later stage though and would in fact constitue a good starting point for someone who wants to start "Moosing" around. :)